lib/cf/broker/src/test/create-test.js

```diff
@@ -1,7 +1,13 @@
 'use strict';
 
+const async = require('async');
 const _ = require('underscore');
 const extend = _.extend;
+
+const request = require('abacus-request');
+const cluster = require('abacus-cluster');
+require.cache[require.resolve('abacus-cluster')].exports =
```

Comment:
Can we move this to before block? Now it is hidden in the require section

This should also allow us to have the broker recreated before each test and isolate the tests.